wcfsetup/install/files/acp/templates/packageUpdateServerList.tpl

```diff
@@ -49,12 +49,15 @@
 						<td class="columnIcon">
 							<span class="icon icon16 fa-{if !$updateServer->isDisabled}check-{/if}square-o jsToggleButton jsTooltip pointer" title="{lang}wcf.global.button.{if !$updateServer->isDisabled}disable{else}enable{/if}{/lang}" data-object-id="{@$updateServer->packageUpdateServerID}"></span>
 							<a href="{link controller='PackageUpdateServerEdit' id=$updateServer->packageUpdateServerID}{/link}" title="{lang}wcf.global.button.edit{/lang}" class="jsTooltip"><span class="icon icon16 fa-pencil"></span></a>
-							<span class="icon icon16 fa-times jsDeleteButton jsTooltip pointer" title="{lang}wcf.global.button.delete{/lang}" data-object-id="{@$updateServer->packageUpdateServerID}" data-confirm-message-html="{lang __encode=true}wcf.acp.updateServer.delete.sure{/lang}"></span>
+							<span class="icon icon16 fa-times {if $updateServer->canDelete()}jsDeleteButton jsTooltip pointer{else}disabled{/if}" title="{lang}wcf.global.button.delete{/lang}" data-object-id="{@$updateServer->packageUpdateServerID}" data-confirm-message-html="{lang __encode=true}wcf.acp.updateServer.delete.sure{/lang}"></span>
 							
 							{event name='itemButtons'}
 						</td>
 						<td class="columnID columnPackageUpdateServerID">{@$updateServer->packageUpdateServerID}</td>
-						<td class="columnTitle columnServerURL"><a href="{link controller='PackageUpdateServerEdit' id=$updateServer->packageUpdateServerID}{/link}" title="{lang}wcf.acp.updateServer.edit{/lang}">{$updateServer->serverURL}</a></td>
+						<td class="columnTitle columnServerURL">
+							<a href="{link controller='PackageUpdateServerEdit' id=$updateServer->packageUpdateServerID}{/link}" title="{lang}wcf.acp.updateServer.edit{/lang}">{$updateServer->serverURL}</a>
+							{if $updateServer->loginUsername}({$updateServer->loginUsername}){/if}
```

Comment:
Not sure about the username, it's a bit lost, at least without providing any meaningful context. At the very least, slap a `wcf.user.username` in front of it or anything else that explains it.

Reply:
I found the information useful. The reason is outlined in the commit message: https://github.com/WoltLab/WCF/pull/3524/commits/d19a9992c39f5dd32d4113dd3309e2f02fceda5d

I'll make this a dedicated column.